stix_shifter/stix_transmission/src/modules/bigfix/bigfix_api_client.py

```diff
@@ -1,8 +1,10 @@
 import base64
 from ..utils.RestApiClient import RestApiClient
 
+TIMEOUT = 60
```

Comment:
The connection timeout is stored on UDS as minutes, not seconds. The default value is 1, but it can always be changed when the data source gets added. If the BigFix API requires seconds for the timeout value, you'll need to do the conversion before creating the client.

Reply:
Hi Danny,
                The BigFix API does not support/provide way to pass timeout parameter.
                We have implemented an internal wait logic in the status call to BigFix API.
                We could convert it to seconds